src/Pixers/SalesManagoAPI/Client.php

```diff
@@ -29,14 +29,21 @@ class Client
 
     /**
      * Initialization.
+     *
+     * @param string $clientId
+     * @param string $endPoint
+     * @param string $apiSecret
+     * @param string $apiKey Basically a random string
```

Comment:
$apiKey is not a random string, someone may misunderstand this comment and in fact provide random string :D

Reply:
@ikselll a czy czasami nie "służący"?